nodogsplash/Makefile

```diff
@@ -7,12 +7,12 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=nodogsplash
 PKG_FIXUP:=autoreconf
-PKG_VERSION:=3.3.0
-PKG_RELEASE:=1
+PKG_VERSION:=3.3.1
+PKG_RELEASE:=1-19.0x
```

Comment:
I think this is a typo.

Reply:
@mwarning 
No, not a typo. See my first comment:

> Known Issue - on OpenWrt >18.x.x, a problem with libmicrohttpd results in
> the gnutls suite being installed causing potential out of memory errors on
> devices <= 64M RAM.
> 
> This version will fail to install on Openwrt 18.06 and 17.01 and is therefore flagged as package release 1-19.0x to facilitate backporting to 18.06 by flagging as 1-18.06 there with the appropriate libmicrohttpd dependency.

There could be other ways of doing this but this will clearly flag the difference between this version and a backported to 10.06 version - actually in the name of the package. This is what PKG_RELEASE is for, is it not?

Reply:
Of course, if pr https://github.com/openwrt/packages/pull/8569 was merged to fix the problem with libmicrohttpd, this would not be necessary.

Reply:
@mwarning 
PR openwrt/packages#8569 has now been merged, so fixes here now removed.